engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/formatter/ConfluentAvroToJsonFormatter.scala

```diff
@@ -39,21 +39,11 @@ private[confluent] class ConfluentAvroToJsonFormatter(schemaRegistryClient: Sche
     printStream.print(Separator)
   }
 
-  // copied from AbstractKafkaAvroDeserializer.deserialize
-  private def readId(bytes: Array[Byte]): Int = {
-    val buffer = getByteBuffer(bytes)
-    buffer.getInt
-  }
-
-  private def getByteBuffer(payload: Array[Byte]) = {
-    val buffer = ByteBuffer.wrap(payload)
-    if (buffer.get != 0)
-      throw new SerializationException("Unknown magic byte!")
-    else
-      buffer
-  }
-
-  // end of copy-paste
+  private def readId(bytes: Array[Byte]): Int =
```

Comment:
Can you move also this to `ConfluentUtils`?

Reply:
Yep